user/tests/readme.md

```diff
@@ -30,6 +36,44 @@ The test applications are based on [Spark customizations](https://github.com/m-m
 library. Please see that repo for details on how to write tests, start and interact
 with the test runner.
 
+## Running Platform Tests
+
+The first step is to flash the test app to a device, e.g.
+
+```
+make TEST=wiring/no_fixture all program-dfu
+```
+
+The device will then restart and connect to the cloud - most tests enable the cloud connection so tha the tests can be monitored remotely. 
```

Comment:
Thanks Kenneth - well spotted! I was typing with my mouth full so couldn't annunciate `that` properly. :-) :-)